ibis/expr/types/generic.py

```diff
@@ -556,9 +556,8 @@ def to_projection(self) -> ir.Table:
         roots = self.op().root_tables()
         if len(roots) > 1:
             raise com.RelationError(
-                'Cannot convert array expression '
-                'involving multiple base table references '
-                'to a projection'
+                'Cannot convert array expression involving multiple base '
```

Comment:
I think these lines weren't covered before and the codecov diff measures the number of changed lines for which there is no coverage introduced.

I'm find with merging this as is.